lib/rules/no-aria-hidden-body.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./base');
+
+const ERROR_MESSAGE =
+  'The aria-hidden attribute should never be present on the <body> element, as it hides the entire document from assistive technology';
+
+module.exports = class NoAriaHiddenBody extends Rule {
+  visitor() {
+    return {
+      ElementNode(node) {
+        let hasAriaHiddenBody =
+          node.tag === 'body' && AstNodeInfo.hasAttribute(node, 'aria-hidden');
+        if (hasAriaHiddenBody) {
+          if (this.mode === 'fix') {
```

Comment:
Could autofixing this violation by deleting the attribute lead to a behavior change? Rule autofixers are generally not supposed to cause behavior changes. Couldn't deleting this attribute be dangerous or require additional manual follow-up to resolve?

Reply:
Screen readers and other assistive technologies don't announce content that's marked as hidden. Applying the aria-hidden="true" attribute to your <body> element hides your entire web page from assistive technology users.
 That's what we are trying to avoid.
https://web.dev/aria-hidden-body/

Reply:
Yes but the question is whether it's completely safe to autofix this or not, or if the user should manually intervene to fix and verify this.

Reply:
I assume it's completely safe to autofix, because nobody with the right mind would want to hide the page from the screenreader, @MelSumner Your thoughts?

Reply:
I'll merge this but we may want to reconsider this.

Reply:
Sure @bmish I am fine with it. Thanks a lot 👍